hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/utils/HddsVolumeUtil.java

```diff
@@ -86,17 +88,29 @@ public static void loadAllHddsVolumeDbStore(MutableVolumeSet hddsVolumeSet,
     // between each HddsVolume -> DbVolume.
     mapDbVolumesToDataVolumesIfNeeded(hddsVolumeSet, dbVolumeSet);
 
-    for (HddsVolume volume : StorageVolumeUtil.getHddsVolumesList(
-        hddsVolumeSet.getVolumesList())) {
-      try {
-        volume.loadDbStore(readOnly);
-      } catch (IOException e) {
-        onFailure(volume);
-        if (logger != null) {
-          logger.error("Load db store for HddsVolume {} failed",
-              volume.getStorageDir().getAbsolutePath(), e);
+    List<CompletableFuture<Void>> futures = new ArrayList<>();
+    List<HddsVolume> hddsVolumes = StorageVolumeUtil.getHddsVolumesList(
+        hddsVolumeSet.getVolumesList());
+    long start = System.currentTimeMillis();
+    for (HddsVolume volume : hddsVolumes) {
+      futures.add(CompletableFuture.runAsync(() -> {
+        try {
```

Comment:
nit: it would be better if you extract out this try/catch to helper function. Will remove too nesting.

```
  private static void loadVolume(HddsVolume volume, boolean readOnly,
                                 Logger logger) {
    try {
      volume.loadDbStore(readOnly);
    } catch (IOException e) {
      onFailure(volume);
      if (logger != null) {
        logger.error("Load db store for HddsVolume {} failed",
            volume.getStorageDir().getAbsolutePath(), e);
      }
    }
  }
```

Reply:
Thanks @hemantk-12, sure.